Syntax error 'constant' in mace.cpp on Windows

##### System information (version)
<!-- Example
- OpenCV => 3.1
- Operating System / Platform => Windows 64 Bit
- Compiler => Visual Studio 2015
-->

- OpenCV => 3.4.1
- Operating System / Platform => Windows 32 bit / Windows 64 bit
- Compiler => Visual Studio 2017

##### Detailed description

Compiler fails with error:
98>E:\Dev\OpenCV\opencv_contrib\modules\face\src\mace.cpp(211): error C2059: syntax error: 'constant'
98>E:\Dev\OpenCV\opencv_contrib\modules\face\src\mace.cpp(212): error C2059: syntax error: 'constant'

This happens because constants rad1 and rad2 are defined in Windows Kits:
c:\Program Files (x86)\Windows Kits\10\Include\10.0.16299.0\um\dlgs.h

Renaming constants does help.

--- a/modules/face/src/mace.cpp
+++ b/modules/face/src/mace.cpp
@@ -208,8 +208,8 @@ struct MACEImpl : MACE {
         re -= m1;
         double value=0;
         double num=0;
-        int rad1=int(floor((double)(45.0/64.0)*(double)IMGSIZE));
-        int rad2=int(floor((double)(27.0/64.0)*(double)IMGSIZE));
+        int rad_1=int(floor((double)(45.0/64.0)*(double)IMGSIZE));
+        int rad_2=int(floor((double)(27.0/64.0)*(double)IMGSIZE));
         // cache a few pow's and sqrts
         std::vector<double> r2(IMGSIZE_2X);
         Mat_<double> radtab(IMGSIZE_2X,IMGSIZE_2X);
@@ -226,8 +226,8 @@ struct MACEImpl : MACE {
         for (int l=0; l<IMGSIZE_2X; l++) {
             for (int m=0; m<IMGSIZE_2X; m++) {
                 double rad = radtab(l,m);
-                if (rad < rad1) {
-                    if (rad > rad2) {
+                if (rad < rad_1) {
+                    if (rad > rad_2) {
                         value += re(l,m);
                         num++;
                     }
@@ -240,8 +240,8 @@ struct MACEImpl : MACE {
         for (int l=0; l<IMGSIZE_2X; l++) {
             for (int m=0; m<IMGSIZE_2X; m++) {
                 double rad = radtab(l,m);
-                if (rad < rad1) {
-                    if (rad > rad2) {
+                if (rad < rad_1) {
+                    if (rad > rad_2) {
                         double d = (value - re(l,m));
                         std2 += d * d;
                     }

